HHS/Head-Start-TTADP
# [TTAHUB-467] apply color palette

## Description of change
UX design did an audit of the colors used in production vs. the original TTA Hub design library and found a considerable amount of discrepancies. This change implements that color palette and also introduces SASS as a way to standardize on color usage.

Because we don't simply use CSS but also use various components that require "CSS-in-JS", I am storing the color values in a JS object, and wrote a very simple node script to compile that file to a SCSS file, which will then be updated throughout the entire project. Hopefully our colors won't be changing that much, and going forward we should only be using colors from this palette

## How to test
Check the site, make sure nothing looks anomalous, review the changes requested in the Jira ticket.

## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-467

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
